doc/tutorials/acrn_configuration_tool.rst

```diff
@@ -202,7 +223,7 @@ Attributes of the ``uos_launcher`` specify the number of User VMs that the curre
   Specify the User VM with its relative ID to Service VM by the "id" attribute.
 
 ``uos_type``:
-  Specify the User VM type, such as  CLEARLINUX, ANDROID, or VXWORKS.
+  Specify the User VM type, such as ``CLEARLINUX``, ``ANDROID``, or ``VXWORKS``.
```

Comment:
I also see other types such as `ALIOS`, `PREEMPT-RT LINUX`, `WINDOWS`, `ZEPHYR` and `GENERIC LINUX`. Shouldn't these be mentionned here for completeness?